test/CMakeLists.txt

```diff
@@ -78,6 +80,18 @@ target_link_libraries(testunitparser lexer test_util config)
 # Use catch_discover instead of add_test for granular test report if CMAKE ver is greater than 3.9,
 # else use the normal add_test method
 # =============================================================================
+set(testvisitor_env "PYTHONPATH=${PROJECT_BINARY_DIR}/lib/python:$ENV{PYTHONPATH}")
+if(NOT LINK_AGAINST_PYTHON)
+  list(APPEND testvisitor_env "NMODL_PYLIB=$ENV{NMODL_PYLIB}")
+  if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+    list(APPEND testvisitor_env
+                "NMODL_WRAPLIB=${PROJECT_BINARY_DIR}/lib/python/nmodl/libpywrapper.so")
+  elseif(CMAKE_SYSTEM_NAME MATCHES "Darwin")
+    list(APPEND testvisitor_env
+                "NMODL_WRAPLIB=${PROJECT_BINARY_DIR}/lib/python/nmodl/libpywrapper.dylib")
```

Comment:
Ahh..I don't need to set these manually?

Reply:
not for testing anymore 😄